includes/frontend/class-wcmp-checkout.php

```diff
@@ -434,6 +453,26 @@ public static function splitShippingMethodString(string $shippingMethod): array
 
         return $split;
     }
+
+    /**
+     * Map keys from the delivery options to the keys used in the adapters.
+     *
+     * @param array $deliveryOptions
+     *
+     * @return array
+     */
+    private static function remapDeliveryOptionsKeys(array $deliveryOptions): array
```

Comment:
getAdapterDeliveryOptionsKeys, getAdapterDeliveryOptions of getDeliveryOptionsForAdapter

Reply:
Hij get niks, how about `convertDeliveryOptionsForAdapter`?

Reply:
Top!